marconi-sidechain/src/Marconi/Sidechain/Api/Routes.hs

```diff
@@ -111,20 +111,25 @@ instance FromJSON GetCurrentSyncedBlockResult where
 
 data GetUtxosFromAddressParams
     = GetUtxosFromAddressParams
-    { queryAddress       :: !String
-    , queryUnspentAtSlot :: !(Maybe Word64)
-    } deriving (Eq, Show)
+    { queryAddress             :: !String -- ^ address to query for
+    , queryCreatedAfterSlotNo  :: !Word64 -- ^ lower bound slotNo to query for
+    , queryUnspentBeforeSlotNo :: !(Maybe Word64) -- ^ optional upper bound slotNo to query for
```

Comment:
It's the other way around. `queryUnspentBeforeSlotNo` is mandatory and the other one is optional given the `API.adoc`